staging/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 15.4.6
+version: 15.5.0
```

Comment:
IMO for forks I think we should stick with patch versions since we're not technically on the next minor version of the upstream chart yet.